web/src/app/rotations/RotationAddUserDialog.tsx

```diff
@@ -32,6 +45,10 @@ const RotationAddUserDialog = (props) => {
     onCompleted: onClose,
   })
 
+  if (error) {
+    return <GenericError error={error.message} />
+  }
```

Comment:
I believe formdialog handles the error so we should be able to leave this out, but then update line 61 to be `nonFieldErrors` instead of `fieldErrors`. The field errors should already be handled by the form container, so we want to make sure any other type of errors get handled by the form dialog